tests/Doctrine/Tests/DBAL/Driver/AbstractSQLServerDriverTest.php

```diff
@@ -54,4 +56,40 @@ protected function getDatabasePlatformsForVersions()
             array('12', 'Doctrine\DBAL\Platforms\SQLServer2012Platform'),
         );
     }
+
+    protected function getDriver() : AbstractSQLServerDriver
+    {
+        return static::createDriver();
+    }
+
+    protected function checkForSkippingTest(AbstractSQLServerDriver $driver) : void
```

Comment:
Please remove this method. The abstract class shouldn't use any knowledge of its descendants.